src/components/Elements.test.tsx

```diff
@@ -139,17 +139,35 @@ describe('Elements', () => {
   });
 
   test('allows a transition from null to a valid Stripe object', () => {
-    let stripeProp: any = null;
-    const wrapper = ({children}: any) => (
-      <Elements stripe={stripeProp}>{children}</Elements>
+    const TestComponent = () => {
```

Comment:
This actually seems too round-about. The way you had it before is fine after I get my head around it. Especially if the old way is the blessed way of testing.

Reply:
Cool, I'll leave it the original way then